cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.ds

```diff
@@ -62,7 +62,12 @@ function createPaymentRequest(args) {
             "value": myAmount
         };
 
-        var paymentMethodType = paymentRequest.paymentMethod.type;
+        if(paymentRequest.paymentMethod.type === "paypal") {
+            var paymentMethodType = paymentRequest.paymentMethod;  //paypal
```

Comment:
why is this different then the rest of the payment methods ?

Reply:
For paypal there's also a `subtype: sdk` expected
https://docs.adyen.com/payment-methods/paypal/web-component#paypal-api

Reply:
this should be provided by the state.data right from the component ?